docs/stack_config.rst

```diff
@@ -348,6 +348,26 @@ Example:
       - !bash "echo hello"
 
 
+``cmd``
+********
+
+Executes string as a shell or PowerShell command.
+
+Syntax:
+
+.. code-block:: yaml
+
+    <hook_point>:
+      - !cmd <bash_command>
```

Comment:
I assume was renamed to show that it support Windows commands....

in which case we should remove `bash` from `<bash_command>`